BUILDER_DEV.md

```diff
@@ -1,390 +1,294 @@
 
-## Pre-Reqs
-1. Use a Linux OS - either native or VM.
-1. Clone the habitat repo to your local filesystem. (If using a VM, you should use the local filesystem for the hab repo instead of using a mounted filesystem, otherwise things may fail later when running services).
-1. The sample commands below use the 'httpie' tool. Install it if not present on your system (https://github.com/jkbrzt/httpie).
-1. A recent version of the habitat cli should also be installed in your dev environment (https://www.habitat.sh/docs/get-habitat/)
-1. Ensure that `sudo -E` functions correctly. Some OSes don't respect the -E flag. If this is the case, remove the `secure_path` section of your sudoers file.
-1. Copy `habitat-builder-dev.2017-10-02.private-key.pem` from 1Password into `/src/.secrets/builder-github-app.pem`
+This document captures the steps to start and run a Builder environment for development. The builder environment includes the builder services, as well as the depot web site.
 
-## Bootstrap the OS with required packages
-You need to make sure you have the required packages installed.
-Run the appropriate shell script under the `support/linux` folder to install the packages.
-Refer to [BUILDING.md](./BUILDING.md) doc for the detailed steps.
+There are several ways of creating a Builder dev environment - but supporting all operating systems and environments has proven to be untenable. This document includes one officially supported way of creating a Builder dev environment, and links to unsupported ways of creating the dev environment that you may use at your own risk.
 
-## Create Builder encryption key pair
-If you haven't already done this earlier, you'll need to create a key pair for builder to encrypt credentials.
-Do this via the `hab user key generate bldr` command. This will generate a key pair with the prefix `bldr`
-in your local `hab/cache/keys` directory. Copy the key pair over to some location that is accessible by the running services.
-You will need to specify that location in the config files below.
+Eventually, we will want to harness the power of the Habitat studio to automate a builder dev environment, but this serves as a stop gap for now.
 
-## Create configuration files
-Some capabilities (such as allowing builder permissions, and turning on auto-building when new packages are uploaded to the depot) require passing in a custom config to the Builder services.
+## Officially Supported Dev Environment
 
-Create the following files somewhere on your local filesystem:
+This Dev Environment involves spinning up a virtual machine (most of the core contributors use VMWare, but Virtual Box should work as well) running Ubuntu 17.10 desktop.
 
-`config_api.toml`
-```toml
-[depot]
-path = "/hab/svc/builder-api/data"
-key_dir = "/path/to/bldr-key-pair"
+### Pre-Reqs
+
+* Ubuntu 17.10 Desktop Virtual Machine
```

Comment:
Server also works with the one caveat about fixing `/etc/apt/sources.list` below.